### ISSUE TITLE
MSStats.csv output file not compatible with MSstats on RStudio

### ISSUE THREAD

**Issue by ChiaraF96** [Nesvilab/FragPipe#1314]
Dear All,

I am currently trying to analyse some data I produced with FragPipe (default workflow), to compare the two modes of acquisition and to write down a protocol for the complete analysis using MSstats on RStudio. I am having some issues using the msstats.csv file produced by FragPipe. It seems like the column format of the .csv file is not matching the format required to use MSstats. I had a conversation with the MSstats team (Devon) and he suggested to ask you about this issue (link to the discussion: https://groups.google.com/u/1/g/msstats/c/bLkdXv_ea9w)

The raw data I am using are originally from this publication: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7898222/.
The peptide digest was independently analyzed by data-dependent acquisition (DDA) and data-independent acquisition (DIA) with the same gradient conditions and the same amounts of sample. A total of eleven DDA technical replicates were acquired; three of them were used for identification purpose only, while the other eight measurements were used for the generation of spectral libraries. A total of three technical replicates were acquired with the DIA method.

For my project, I analysed  with FragPipe 3 DDA and 3 DIA technical replicates.

FragPipe's version used: 

```
$ /home/cfusco/src/FragPipe-20.0/bin/fragpipe --help
FragPipe v20.0
(c) University of Michigan
System OS: Linux, Architecture: amd64
Java Info: 17.0.8, OpenJDK 64-Bit Server VM, Red Hat, Inc.
```

R and MSstats versions used:

```
> sessionInfo()
R version 4.3.1 (2023-06-16)
Platform: x86_64-redhat-linux-gnu (64-bit)
Running under: Fedora Linux 38 (Workstation Edition)

Matrix products: default
BLAS/LAPACK: FlexiBLAS OPENBLAS-OPENMP;  LAPACK version 3.11.0

locale:
 [1] LC_CTYPE=en_US.UTF-8       LC_NUMERIC=C              
 [3] LC_TIME=en_US.UTF-8        LC_COLLATE=en_US.UTF-8    
 [5] LC_MONETARY=en_US.UTF-8    LC_MESSAGES=en_US.UTF-8   
 [7] LC_PAPER=en_US.UTF-8       LC_NAME=C                 
 [9] LC_ADDRESS=C               LC_TELEPHONE=C            
[11] LC_MEASUREMENT=en_US.UTF-8 LC_IDENTIFICATION=C       

time zone: Europe/Amsterdam
tzcode source: system (glibc)

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods  
[7] base     

other attached packages:
[1] MSstats_4.10.0

loaded via a namespace (and not attached):
 [1] Matrix_1.6-1.1        preprocessCore_1.64.0 gtable_0.3.4         
 [4] limma_3.58.0          compiler_4.3.1        gtools_3.9.4         
 [7] Rcpp_1.0.11           bitops_1.0-7          MSstatsConvert_1.12.0
[10] splines_4.3.1         scales_1.2.1          boot_1.3-28.1        
[13] statmod_1.5.0         lattice_0.22-5        ggplot2_3.4.4        
[16] R6_2.5.1              backports_1.4.1       MASS_7.3-60          
[19] ggrepel_0.9.4         checkmate_2.3.0       tibble_3.2.1         
[22] nloptr_2.0.3          munsell_0.5.0         minqa_1.2.6          
[25] pillar_1.9.0          rlang_1.1.1           utf8_1.2.4           
[28] marray_1.80.0         caTools_1.18.2        cli_3.6.1            
[31] magrittr_2.0.3        grid_4.3.1            lme4_1.1-34          
[34] lifecycle_1.0.3       nlme_3.1-163          vctrs_0.6.4          
[37] KernSmooth_2.23-22    glue_1.6.2            data.table_1.14.8    
[40] log4r_0.4.3           survival_3.5-7        fansi_1.0.5          
[43] colorspace_2.1-0      tools_4.3.1           pkgconfig_2.0.3      
[46] gplots_3.1.3   

```

Below is the content of my "experiment1" folder with outputs, manifest, parameters and the msstats.csv file.

```
$ exa -lRT -L 4 /home/cfusco/data/experiment1
drwxr-xr-x@    - cfusco 30 Oct 11:41 /home/cfusco/data/experiment1
drwxr-xr-x@    - cfusco 25 Oct 10:15 ├── config1
.rw-r--r--@  431 cfusco 23 Sep 08:16 │  ├── experiment1manifest.fp-manifest
.rw-r--r--@ 9.7k cfusco 23 Sep 08:20 │  ├── fragger.params
drwxr-xr-x@    - cfusco 25 Oct 11:34 │  ├── results1
.rw-r--r--@  28M cfusco 23 Sep 21:24 │  │  ├── combined.prot.xml
.rw-r--r--@ 830k cfusco 23 Sep 21:25 │  │  ├── combined_protein.tsv
drwxr-xr-x@    - cfusco 23 Sep 21:25 │  │  ├── control_1
.rw-r--r--@ 310k cfusco 23 Sep 12:47 │  │  │  ├── diaumpire_se.log
.rw-r--r--@ 2.3k cfusco 23 Sep 21:25 │  │  │  ├── filter.log
.rw-r--r--@  80M cfusco 23 Sep 21:24 │  │  │  ├── interact-Lysate_DDA_Rep1_ID.pep.xml
.rw-r--r--@ 3.2M cfusco 23 Sep 21:25 │  │  │  ├── ion.tsv
.rw-r--r--@  63M cfusco 23 Sep 21:21 │  │  │  ├── Lysate_DDA_Rep1_ID.pepXML
.rw-r--r--@  15M cfusco 23 Sep 21:21 │  │  │  ├── Lysate_DDA_Rep1_ID.pin
.rw-r--r--@  17M cfusco 23 Sep 21:23 │  │  │  ├── Lysate_DDA_Rep1_ID_edited.pin
.rw-r--r--@ 100M cfusco 23 Sep 12:47 │  │  │  ├── Lysate_DIA_Rep1_ID_Q1.mzML
.rw-r--r--@  81M cfusco 23 Sep 12:47 │  │  │  ├── Lysate_DIA_Rep1_ID_Q2.mzML
.rw-r--r--@  53M cfusco 23 Sep 12:47 │  │  │  ├── Lysate_DIA_Rep1_ID_Q3.mzML
drwxr-xr-x@    - cfusco 23 Sep 21:23 │  │  │  ├── MSBooster_RTplots
.rw-r--r--@ 2.2M cfusco 23 Sep 21:25 │  │  │  ├── peptide.tsv
.rw-r--r--@ 1.8M cfusco 23 Sep 21:25 │  │  │  ├── protein.fas
.rw-r--r--@ 581k cfusco 23 Sep 21:25 │  │  │  ├── protein.tsv
.rw-r--r--@  20M cfusco 23 Sep 21:25 │  │  │  ├── psm.tsv
.rw-r--r--@  13M cfusco 23 Sep 21:23 │  │  │  ├── spectraRT.predicted.bin
.rw-r--r--@ 1.9M cfusco 23 Sep 21:22 │  │  │  ├── spectraRT.tsv
.rw-r--r--@ 1.9M cfusco 23 Sep 21:22 │  │  │  └── spectraRT_full.tsv
drwxr-xr-x@    - cfusco 23 Sep 21:25 │  │  ├── control_1_1
.rw-r--r--@ 2.6k cfusco 23 Sep 21:25 │  │  │  ├── filter.log
.rw-r--r--@  16M cfusco 23 Sep 21:24 │  │  │  ├── interact-Lysate_DIA_Rep1_ID_Q1.pep.xml
.rw-r--r--@ 4.4M cfusco 23 Sep 21:24 │  │  │  ├── interact-Lysate_DIA_Rep1_ID_Q2.pep.xml
.rw-r--r--@ 6.9M cfusco 23 Sep 21:24 │  │  │  ├── interact-Lysate_DIA_Rep1_ID_Q3.pep.xml
.rw-r--r--@ 2.8M cfusco 23 Sep 21:25 │  │  │  ├── ion.tsv
.rw-r--r--@  14M cfusco 23 Sep 21:21 │  │  │  ├── Lysate_DIA_Rep1_ID_Q1.pepXML
.rw-r--r--@ 3.4M cfusco 23 Sep 21:21 │  │  │  ├── Lysate_DIA_Rep1_ID_Q1.pin
.rw-r--r--@ 3.9M cfusco 23 Sep 21:23 │  │  │  ├── Lysate_DIA_Rep1_ID_Q1_edited.pin
.rw-r--r--@ 7.0M cfusco 23 Sep 21:21 │  │  │  ├── Lysate_DIA_Rep1_ID_Q2.pepXML
.rw-r--r--@ 1.7M cfusco 23 Sep 21:21 │  │  │  ├── Lysate_DIA_Rep1_ID_Q2.pin
.rw-r--r--@ 1.9M cfusco 23 Sep 21:23 │  │  │  ├── Lysate_DIA_Rep1_ID_Q2_edited.pin
.rw-r--r--@ 7.1M cfusco 23 Sep 21:21 │  │  │  ├── Lysate_DIA_Rep1_ID_Q3.pepXML
.rw-r--r--@ 1.7M cfusco 23 Sep 21:21 │  │  │  ├── Lysate_DIA_Rep1_ID_Q3.pin
.rw-r--r--@ 1.9M cfusco 23 Sep 21:23 │  │  │  ├── Lysate_DIA_Rep1_ID_Q3_edited.pin
drwxr-xr-x@    - cfusco 23 Sep 21:23 │  │  │  ├── MSBooster_RTplots
.rw-r--r--@ 2.0M cfusco 23 Sep 21:25 │  │  │  ├── peptide.tsv
.rw-r--r--@ 1.8M cfusco 23 Sep 21:25 │  │  │  ├── protein.fas
.rw-r--r--@ 592k cfusco 23 Sep 21:25 │  │  │  ├── protein.tsv
.rw-r--r--@ 6.7M cfusco 23 Sep 21:25 │  │  │  └── psm.tsv
drwxr-xr-x@    - cfusco 23 Sep 21:25 │  │  ├── control_2
.rw-r--r--@ 310k cfusco 23 Sep 17:08 │  │  │  ├── diaumpire_se.log
.rw-r--r--@ 2.4k cfusco 23 Sep 21:25 │  │  │  ├── filter.log
.rw-r--r--@  81M cfusco 23 Sep 21:24 │  │  │  ├── interact-Lysate_DDA_Rep2_ID.pep.xml
.rw-r--r--@ 3.5M cfusco 23 Sep 21:25 │  │  │  ├── ion.tsv
.rw-r--r--@  63M cfusco 23 Sep 21:21 │  │  │  ├── Lysate_DDA_Rep2_ID.pepXML
.rw-r--r--@  15M cfusco 23 Sep 21:21 │  │  │  ├── Lysate_DDA_Rep2_ID.pin
.rw-r--r--@  17M cfusco 23 Sep 21:23 │  │  │  ├── Lysate_DDA_Rep2_ID_edited.pin
.rw-r--r--@ 104M cfusco 23 Sep 17:08 │  │  │  ├── Lysate_DIA_Rep2_ID_Q1.mzML
.rw-r--r--@  83M cfusco 23 Sep 17:08 │  │  │  ├── Lysate_DIA_Rep2_ID_Q2.mzML
.rw-r--r--@  59M cfusco 23 Sep 17:08 │  │  │  ├── Lysate_DIA_Rep2_ID_Q3.mzML
drwxr-xr-x@    - cfusco 23 Sep 21:23 │  │  │  ├── MSBooster_RTplots
.rw-r--r--@ 2.4M cfusco 23 Sep 21:25 │  │  │  ├── peptide.tsv
.rw-r--r--@ 1.8M cfusco 23 Sep 21:25 │  │  │  ├── protein.fas
.rw-r--r--@ 577k cfusco 23 Sep 21:25 │  │  │  ├── protein.tsv
.rw-r--r--@  20M cfusco 23 Sep 21:25 │  │  │  └── psm.tsv
drwxr-xr-x@    - cfusco 23 Sep 21:25 │  │  ├── control_2_2
.rw-r--r--@ 2.6k cfusco 23 Sep 21:25 │  │  │  ├── filter.log
.rw-r--r--@  17M cfusco 23 Sep 21:24 │  │  │  ├── interact-Lysate_DIA_Rep2_ID_Q1.pep.xml
.rw-r--r--@ 4.3M cfusco 23 Sep 21:24 │  │  │  ├── interact-Lysate_DIA_Rep2_ID_Q2.pep.xml
.rw-r--r--@ 7.6M cfusco 23 Sep 21:24 │  │  │  ├── interact-Lysate_DIA_Rep2_ID_Q3.pep.xml
.rw-r--r--@ 2.9M cfusco 23 Sep 21:25 │  │  │  ├── ion.tsv
.rw-r--r--@  15M cfusco 23 Sep 21:21 │  │  │  ├── Lysate_DIA_Rep2_ID_Q1.pepXML
.rw-r--r--@ 3.5M cfusco 23 Sep 21:21 │  │  │  ├── Lysate_DIA_Rep2_ID_Q1.pin
.rw-r--r--@ 4.1M cfusco 23 Sep 21:23 │  │  │  ├── Lysate_DIA_Rep2_ID_Q1_edited.pin
.rw-r--r--@ 7.3M cfusco 23 Sep 21:21 │  │  │  ├── Lysate_DIA_Rep2_ID_Q2.pepXML
.rw-r--r--@ 1.7M cfusco 23 Sep 21:21 │  │  │  ├── Lysate_DIA_Rep2_ID_Q2.pin
.rw-r--r--@ 2.0M cfusco 23 Sep 21:23 │  │  │  ├── Lysate_DIA_Rep2_ID_Q2_edited.pin
.rw-r--r--@ 7.8M cfusco 23 Sep 21:21 │  │  │  ├── Lysate_DIA_Rep2_ID_Q3.pepXML
.rw-r--r--@ 1.9M cfusco 23 Sep 21:21 │  │  │  ├── Lysate_DIA_Rep2_ID_Q3.pin
.rw-r--r--@ 2.1M cfusco 23 Sep 21:23 │  │  │  ├── Lysate_DIA_Rep2_ID_Q3_edited.pin
drwxr-xr-x@    - cfusco 23 Sep 21:23 │  │  │  ├── MSBooster_RTplots
.rw-r--r--@ 2.1M cfusco 23 Sep 21:25 │  │  │  ├── peptide.tsv
.rw-r--r--@ 1.9M cfusco 23 Sep 21:25 │  │  │  ├── protein.fas
.rw-r--r--@ 599k cfusco 23 Sep 21:25 │  │  │  ├── protein.tsv
.rw-r--r--@ 7.0M cfusco 23 Sep 21:25 │  │  │  └── psm.tsv
drwxr-xr-x@    - cfusco 23 Sep 21:25 │  │  ├── control_3
.rw-r--r--@ 310k cfusco 23 Sep 21:17 │  │  │  ├── diaumpire_se.log
.rw-r--r--@ 2.4k cfusco 23 Sep 21:25 │  │  │  ├── filter.log
.rw-r--r--@  86M cfusco 23 Sep 21:24 │  │  │  ├── interact-Lysate_DDA_Rep3_ID.pep.xml
.rw-r--r--@ 3.7M cfusco 23 Sep 21:25 │  │  │  ├── ion.tsv
.rw-r--r--@  68M cfusco 23 Sep 21:22 │  │  │  ├── Lysate_DDA_Rep3_ID.pepXML
.rw-r--r--@  16M cfusco 23 Sep 21:22 │  │  │  ├── Lysate_DDA_Rep3_ID.pin
.rw-r--r--@  18M cfusco 23 Sep 21:23 │  │  │  ├── Lysate_DDA_Rep3_ID_edited.pin
.rw-r--r--@  95M cfusco 23 Sep 21:17 │  │  │  ├── Lysate_DIA_Rep3_ID_Q1.mzML
.rw-r--r--@  77M cfusco 23 Sep 21:17 │  │  │  ├── Lysate_DIA_Rep3_ID_Q2.mzML
.rw-r--r--@  52M cfusco 23 Sep 21:17 │  │  │  ├── Lysate_DIA_Rep3_ID_Q3.mzML
drwxr-xr-x@    - cfusco 23 Sep 21:23 │  │  │  ├── MSBooster_RTplots
.rw-r--r--@ 2.5M cfusco 23 Sep 21:25 │  │  │  ├── peptide.tsv
.rw-r--r--@ 1.8M cfusco 23 Sep 21:25 │  │  │  ├── protein.fas
.rw-r--r--@ 591k cfusco 23 Sep 21:25 │  │  │  ├── protein.tsv
.rw-r--r--@  21M cfusco 23 Sep 21:25 │  │  │  └── psm.tsv
drwxr-xr-x@    - cfusco 23 Sep 21:25 │  │  ├── control_3_3
.rw-r--r--@ 2.6k cfusco 23 Sep 21:25 │  │  │  ├── filter.log
.rw-r--r--@  15M cfusco 23 Sep 21:24 │  │  │  ├── interact-Lysate_DIA_Rep3_ID_Q1.pep.xml
.rw-r--r--@ 4.1M cfusco 23 Sep 21:24 │  │  │  ├── interact-Lysate_DIA_Rep3_ID_Q2.pep.xml
.rw-r--r--@ 6.5M cfusco 23 Sep 21:24 │  │  │  ├── interact-Lysate_DIA_Rep3_ID_Q3.pep.xml
.rw-r--r--@ 2.6M cfusco 23 Sep 21:25 │  │  │  ├── ion.tsv
.rw-r--r--@  13M cfusco 23 Sep 21:21 │  │  │  ├── Lysate_DIA_Rep3_ID_Q1.pepXML
.rw-r--r--@ 3.2M cfusco 23 Sep 21:21 │  │  │  ├── Lysate_DIA_Rep3_ID_Q1.pin
.rw-r--r--@ 3.7M cfusco 23 Sep 21:23 │  │  │  ├── Lysate_DIA_Rep3_ID_Q1_edited.pin
.rw-r--r--@ 6.9M cfusco 23 Sep 21:21 │  │  │  ├── Lysate_DIA_Rep3_ID_Q2.pepXML
.rw-r--r--@ 1.6M cfusco 23 Sep 21:21 │  │  │  ├── Lysate_DIA_Rep3_ID_Q2.pin
.rw-r--r--@ 1.9M cfusco 23 Sep 21:23 │  │  │  ├── Lysate_DIA_Rep3_ID_Q2_edited.pin
.rw-r--r--@ 7.0M cfusco 23 Sep 21:21 │  │  │  ├── Lysate_DIA_Rep3_ID_Q3.pepXML
.rw-r--r--@ 1.6M cfusco 23 Sep 21:21 │  │  │  ├── Lysate_DIA_Rep3_ID_Q3.pin
.rw-r--r--@ 1.9M cfusco 23 Sep 21:23 │  │  │  ├── Lysate_DIA_Rep3_ID_Q3_edited.pin
drwxr-xr-x@    - cfusco 23 Sep 21:23 │  │  │  ├── MSBooster_RTplots
.rw-r--r--@ 1.9M cfusco 23 Sep 21:25 │  │  │  ├── peptide.tsv
.rw-r--r--@ 1.8M cfusco 23 Sep 21:25 │  │  │  ├── protein.fas
.rw-r--r--@ 576k cfusco 23 Sep 21:25 │  │  │  ├── protein.tsv
.rw-r--r--@ 6.3M cfusco 23 Sep 21:25 │  │  │  └── psm.tsv
.rw-r--r--@ 1.1k cfusco 23 Sep 08:26 │  │  ├── filelist_proteinprophet.txt
.rw-r--r--@  85M cfusco 23 Sep 21:28 │  │  ├── FP-PDV.db
.rw-r--r--@ 9.7k cfusco 23 Sep 08:26 │  │  ├── fragger_dda.params
.rw-r--r--@  431 cfusco 23 Sep 21:25 │  │  ├── fragpipe-files.fp-manifest
.rw-r--r--@  11k cfusco 23 Sep 21:25 │  │  ├── fragpipe.workflow
.rw-r--r--@ 1.0M cfusco 23 Sep 21:25 │  │  ├── log_2023-09-23_15-25-52.txt
.rw-r--r--@ 1.0M cfusco 23 Sep 21:27 │  │  ├── log_2023-09-23_15-26-55.txt
.rw-r--r--@  943 cfusco 23 Sep 08:26 │  │  ├── msbooster_params.txt
.rw-r--r--@  40M cfusco 23 Sep 21:25 │  │  ├── msstats.csv
.rw-r--r--@ 118k cfusco 23 Sep 21:25 │  │  ├── reprint.spc.tsv
.rw-r--r--@ 2.7k cfusco 23 Sep 08:26 │  │  └── umpire-se.params

```

To analyse the msstats.csv file, I followed the FragPipe tutorial: https://fragpipe.nesvilab.org/docs/tutorial_msstats.html

RStudio output with errors:

```
> rm(list = ls())
> library(stringr)
> library(readr)
> library(MSstats)

Attaching package: ‘MSstats’

The following object is masked from ‘package:grDevices’:

    savePlot

> rootDir <- "/home/cfusco/data/experiment1/config1/results1/" # Specify the path of the directory containing MSstats.csv.
> print(str_c("Using IonQuant's result from experiment1/config1/results1", rootDir))
[1] "Using IonQuant's result from experiment1/config1/results1/home/cfusco/data/experiment1/config1/results1/"
> raw <- read_csv(str_c(rootDir, "msstats.csv"), na = c("", "NA", "0"))
Rows: 212906 Columns: 17                                                                                                                               
── Column specification ───────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
Delimiter: ","
chr (11): Spectrum.Name, Spectrum.File, Peptide.Sequence, Modified.Peptide.Sequence, Gene, Mapped.Genes, Protein, Protein.ID, Mapped.Proteins, Prot...
dbl  (5): Probability, Charge, Protein.Start, Protein.End, Purity
lgl  (1): Is.Unique

ℹ Use `spec()` to retrieve the full column specification for this data.
ℹ Specify the column types or set `show_col_types = FALSE` to quiet this message.
Warning message:
One or more parsing issues, call `problems()` on your data frame for details, e.g.:
  dat <- vroom(...)
  problems(dat) 
> raw$ProteinName <- factor(raw$ProteinName)
Error in `$<-`:
! Assigned data `factor(raw$ProteinName)` must be compatible with existing data.
✖ Existing data has 212906 rows.
✖ Assigned data has 0 rows.
ℹ Only vectors of size 1 are recycled.
Caused by error in `vectbl_recycle_rhs_rows()`:
! Can't recycle input of size 0 to size 212906.
Run `rlang::last_trace()` to see where the error occurred.
Warning message:
Unknown or uninitialised column: `ProteinName`. 
> setwd(rootDir)
> processedData <- dataProcess(raw, logTrans = 10)
Error in .local(input, ...) : 
  Missing columns in the input: ProteinName PeptideSequence PeptideModifiedSequence PrecursorCharge FragmentIon ProductCharge IsotopeLabelType Condition BioReplicate Run

```

I also did a second default analysis with only 3 DDA files and the error on MSstats was still there.

Do you have an idea of what could cause this issue with the MSstats.csv output file?
Attached you can find the log of the analysis.

Thank you in advance,
Chiara.

[log_2023-09-23_15-25-52.txt](https://github.com/Nesvilab/FragPipe/files/13216426/log_2023-09-23_15-25-52.txt)

**Comment by fcyu** [comment]
Hi Chiara,

First of all, if there are DIA data, you should not assign any experiment group for the current FragPipe version.
Second, if there are DIA data, both the DDA and DIA data will be used for the spectral library building, and then the DIA data will be used for quantification.
Third, for DDA LFQ or DIA, you should **uncheck** the "generate MSstats files" box in the "FDR Filter and Report" panel because that is only for TMT (I know it is a little confusion. We might move that box to somewhere else). In the current version, IonQuant can generate MSstats for DDA LFQ. The next version, we will have the DIA MSstats support.

Best,

Fengchao

**Comment by ChiaraF96** [comment]
Dear Fengchao,

Thank you for your quick reply and the valuable informations.
I will have to find an alternative way to analyse the DIA.

Regarding the DDA, I just tried running the analysis several times using only the 3 DDA files. I tried different combinations and these were the results:

- 3 DDA files without specifying experiment number or bioreplicate and with "generate MSstats files" box UNCHECKED : no msstats file was generated from the analysis.
- 3 DDA files specifying only experiment number (number 1 for each of the 3 DDA files) and with "generate MSstats files" box UNCHECKED : no msstats file was generated from the analysis.
making sure that the "generate MSstats files" was unchecked.
- 3 DDA files without specifying experiment number or bioreplicate and with "generate MSstats files" box CHECKED : msstats file was generated from the analysis but with the wrong format and the same issue with MSstats.
- 3 DDA files specifying experiment number  (exp number 1 for each of the 3 files) AND bioreplicate (biorep 1,2, and 3) and with "generate MSstats files" box UNCHECKED : msstats file was generate from the analysis but with the wrong format and the same issue with MSStats.

Nothing else about the workflow and the different tabs was changed.

What else could I try? 


Best,
Chiara
